community/emacs/sources

```diff
@@ -0,0 +1 @@
+https://gnuftp.uib.no/emacs/emacs-26.3.tar.xz
```

Comment:
Shouldn't this be either the [main GNU ftp](https://ftp.gnu.org/gnu/emacs/emacs-26.3.tar.xz) or an auto-redirect using http://ftpmirror.gnu.org/emacs/?
I don't think it makes sense to use one mirror for every KISS download.

Reply:
You're right. I'll take "your" first link...